README.md

```diff
@@ -67,19 +67,24 @@ const markdownStyle: MarkdownStyle = {
   h1: {
     fontSize: 25,
   },
+  emoji: {
+    fontSize: 20,
+  },
   blockquote: {
     borderColor: 'gray',
     borderWidth: 6,
     marginLeft: 6,
     paddingLeft: 6,
   },
   code: {
-    fontFamily: 'monospace',
+    fontFamily: FONT_FAMILY_MONOSPACE,
```

Comment:
Please add `FONT_FAMILY_MONOSPACE` variable definition

Reply:
Added in 874641c